Sources/llbuild2fx/Key.swift

```diff
@@ -182,30 +226,39 @@ final class FXFunction<K: FXKey>: LLBTypedCachingFunction<InternalKey<K>, Intern
     > {
         let actualKey = key.key
 
+        let fxfi = FXFunctionInterface(actualKey, fi)
+
+        let keyData = try! FXEncoder().encode(actualKey)
+        let encodedKey = String(bytes: keyData, encoding: .utf8)!
+        let keyPrefix = FXCacheKeyPrefixMemoizer.get(for: actualKey)
+
         ctx.fxBuildEngineStats.add(key: key.name)
         let span = ctx.tracer?.startSpan(
-            key.logDescription(),
-            context: .TODO(),
+            keyPrefix,
```

Comment:
How does this memoized value compare to the previous `logDescription` value?

Reply:
`logDescription` evaluates to `"KEY: //\(cachePath) [HASH: \(hashValue)]"`.. The `hashValue` was just visual noise. Now it emits just the key name (without args).